openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -433,10 +433,7 @@ OpenAssessment.ResponseView.prototype = {
         * rejected if the user cancels the submission
     **/
     confirmSubmission: function() {
-        var msg = (
-            "You're about to submit your response for this assignment. " +
-            "After you submit this response, you can't change it or submit a new response."
-        );
+        var msg = gettext("You're about to submit your response for this assignment. After you submit this response, you can't change it or submit a new response.");
```

Comment:
Ah, this makes sense. I think it would be good to add a comment above this line so future developers don't make the same mistake.